### ISSUE TITLE
Deprication warning in sionna metrics

### ISSUE THREAD

**Issue by danielschaeufele** [NVlabs/sionna#14]
**Describe the bug**
`BitwiseMutualInformation` and `BitErrorRate` implement the deprecated `reset_states()` method. The method should be renamed to `reset_state()`.

**To Reproduce**
Execute the following code:

```
from sionna.utils import BitwiseMutualInformation, BitErrorRate

bmi = BitwiseMutualInformation()
bmi.reset_state()

ber = BitErrorRate()
ber.reset_state()
```

The following warnings will appear:

```
UserWarning: Metric BitwiseMutualInformation implements a `reset_states()` method; rename it to `reset_state()` (without the final "s"). The name `reset_states()` has been deprecated to improve API consistency.
  bmi.reset_state()
UserWarning: Metric BitErrorRate implements a `reset_states()` method; rename it to `reset_state()` (without the final "s"). The name `reset_states()` has been deprecated to improve API consistency.
  ber.reset_state()
```

**Expected behavior**
No warning should appear

**System:**
 - Python version: 3.10.4
 - TensorFlow version: 2.8.0

**Comment by SebastianCa** [comment]
Thank you for reporting this issue. We could reproduce the deprecated warning and will fix it in the next Sionna release.

**Comment by gmarcusm** [comment]
Thanks, fixed in v0.9.1
